.github/workflows/publish-to-pypi.yml

```diff
@@ -11,20 +14,23 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # Fetched tags too
+
```

Comment:
This should not be needed, because pipeline runs on tag only.

Reply:
I'm unsure. It runs on the tag but I'm not sure if that means the tag is actually checked out too. In any case, it won't hurt.